app/models/iiif_harvester.rb

```diff
@@ -34,6 +34,16 @@ def tei_url
     tei_template_url.gsub('{shelfmark}', shelfmark)
   end
 
+  def thumbnails
+    return [] if manifest['sequences'].blank?
+
+    manifest['sequences'].map do |sequence|
+      next if sequence['canvases'].blank?
+
+      sequence['canvases'].first.dig('thumbnail', '@id')
```

Comment:
Thinking about this more, this will most likely only ever be one.  Should we index the thumbnail of every canvas w/i a sequence?

Reply:
I've updated this to get all thumbnails in all canvases in all sequences.